src/js/data-model.jsx

```diff
@@ -103,13 +123,30 @@ var DataModel = React.createClass({
         }
 
         var dqaTable = this.props.dqa && this.props.dqa[this.props.activeTable];
-        var dqaField = dqaTable && dqaTable.fields[this.props.activeField];
-
-        if (dqaTable !== undefined && dqaField !== undefined) {
+        var dqaField = dqaTable && dqaTable[this.props.activeField];
+
+        if (dqaTable !== undefined) {
```

Comment:
Since we know these can't be falsy values in general.. doing `if (!dqaTable)` would be sufficient here.